lib/vttcue.js

```diff
@@ -68,27 +99,28 @@
     var _size = 50;
     var _align = "middle";
 
-    Object.defineProperties(this, {
-      "id": {
-        enumerable: true,
+    Object.defineProperty(cue,
```

Comment:
Yes, exactly right. IE8 doesn't support enumerable, so, we set up the baseObj with that property depending on whether we are in IE8 or not.